libs/ui/src/invalid_card_screen.tsx

```diff
@@ -71,7 +71,7 @@ export function InvalidCardScreen({
             return 'election manager card';
           case 'poll_worker':
             return 'poll worker card';
-          /* istanbul ignore next */
+          /* istanbul ignore next - @preserve */
```

Comment:
Does this need to be moved inside the `default` block?

Reply:
[c97fcfb](https://github.com/votingworks/vxsuite/pull/6032/commits/c97fcfb9b4fc5a423630c2a45863d37c7dff06d8)